src/shared-components/dropdown/index.tsx

```diff
@@ -45,15 +46,17 @@ type DropdownProps<T> = {
  * This ships with a mobile implementation that will handle mobile devices automatically.
  */
 export const Dropdown = <T extends OptionType>({
-  borderRadius = '4px',
+  borderRadius,
   onChange,
   options,
   optionsContainerMaxHeight = '250px',
   textAlign = 'left',
   value,
 }: DropdownProps<T>) => {
+  const theme = useTheme();
   const [isOpen, setIsOpen] = useState(false);
   const touchSupported = 'ontouchstart' in document.documentElement;
+  const borderRadiusValue = borderRadius || theme.BORDER_RADIUS.small;
```

Comment:
My previous comments about preferring to keep the logic in `styled` components doesn't apply here, since we wouldn't want to duplicate ourselves 🙂